usr/src/uts/common/fs/zfs/sys/dsl_pool.h

```diff
@@ -122,6 +122,8 @@ typedef struct dsl_pool {
 	txg_list_t dp_dirty_dirs;
 	txg_list_t dp_sync_tasks;
 	taskq_t *dp_sync_taskq;
+	taskq_t *dp_zil_clean_taskq;
+	txg_list_t dp_early_sync_tasks;
```

Comment:
`dp_early_sync_tasks` is not used?

Reply:
Good catch. That's not supposed to be there. I must have mis-merged when pulling this over from Delphix to illumos and didn't notice that. Thanks @chrisrd, I'll open a PR to remove this.

Reply:
please see #455 .. thanks for noticing, and sorry for introducing the error to begin with.